File: jupyter2/problems/parametric/function_in_parameters/jl/function_in_parameters.jl
### begin my answer

### end my answer

function main()
    @assert(app1(f) == 1.0)
    @assert(app1(g) == [1.0])
    @assert(app(f, 4) == 4)
    @assert(app(h, ["bye"]) == "bye")
    println("OK")
end

main()

File: jupyter2/problems/parametric/function_in_parameters/rs/function_in_parameters.rs
/** begin my answer */

/** end my answer */

fn main() {
    assert!(app1(f) == 1.0);
    assert!(app1(g) == [1.0]);
    assert!(app(f, 4) == 4);
    assert!(app(h, &vec!["bye"]) == Some(&"bye"));
    println!("OK")
}
